### PR TITLE
docs: readthedocs config

### PR DESCRIPTION
**Pull request by mikeldking**

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    tools:
+        python: "3.12"
+        # You can also specify other tool versions:
+        # nodejs: "19"
+        # rust: "1.64"
+        # golang: "1.19"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+    configuration: docs/conf.py
```
Comment (axiomofjoy) [inline .readthedocs.yaml:20]: Not sure we have this file?

Comment (axiomofjoy) [reply .readthedocs.yaml:20]: We have a conflict in the `docs` directory.

Comment (mikeldking) [reply .readthedocs.yaml:20]: yeah I'm switching it to be api_docs

**Review by axiomofjoy — APPROVED** [review]

File: .readthedocs.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    tools:
+        python: "3.12"
+        # You can also specify other tool versions:
+        # nodejs: "19"
+        # rust: "1.64"
+        # golang: "1.19"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+    configuration: api_docs/conf.py
```
Comment (mikeldking) [inline .readthedocs.yaml:20]: cc @cjunkin for whatever we settle on

**Comment by cjunkin** [comment]
I have read the CLA Document and I hereby sign the CLA

File: api_docs.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd docs/api_sphinx
+sphinx-apidoc -o ./docs_sphinx/source ./src/phoenix/inferences -f
+sphinx-apidoc -o ./docs_sphinx/source ./src/phoenix/session -f
+sphinx-apidoc -o ./docs_sphinx/source ./src/phoenix/evals -f
```
Comment (axiomofjoy) [inline api_docs.sh:6]: This sub-module is legacy. We can exclude.

File: .readthedocs.yaml
Comment (axiomofjoy) [inline .readthedocs.yaml:1]: Let's clean up the unnecessary comments in this file.

File: docs/api_sphinx/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../../"))
+sys.path.insert(0, os.path.abspath("../../src/phoenix"))
+# sys.path.insert(0, os.path.abspath('../../packages/phoenix-evals/src/phoenix/evals'))
```
Comment (axiomofjoy) [inline docs/api_sphinx/source/conf.py:6]: Can we avoid this if we install Phoenix?

File: docs/api_sphinx/source/evals.models.rst
Comment (axiomofjoy) [inline docs/api_sphinx/source/evals.models.rst:1]: Possible to use markdown rather that rst?

File: docs/api_sphinx/source/index.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.. Phoenix documentation master file, created by
+   sphinx-quickstart on Wed Jun 26 16:05:49 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
```
Comment (axiomofjoy) [inline docs/api_sphinx/source/index.rst:4]: Can nix comment.

File: .readthedocs.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  tools:
+    python: "3.8"
+    # You can also specify other tool versions:
+    # nodejs: "20"
+    # rust: "1.70"
+    # golang: "1.20"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+  configuration: docs/api_sphinx/source/conf.py
```
Comment (axiomofjoy) [inline .readthedocs.yaml:19]: I think we probably want to move out of `docs`. The reason being that this directory is managed by gitbook.

**Review by axiomofjoy — COMMENTED** [review]
Let's nix the auto-generated comments unless they're useful.

**Pull request merged**